components/prism-graphql.js

```diff
@@ -11,7 +11,7 @@ Prism.languages.graphql = {
 		pattern: /@[a-z_]\w*/i,
 		alias: 'function'
 	},
-	'attr-name': /[a-z_]\w*(?=\s*:)/i,
+	'attr-name': /[a-z_]\w*((?=\s*:)|(?=\s*\([^\)]*\):))/i,
```

Comment:
2 nits:

1. Please use `[^()]` inside the parentheses.
2. I think the lookahead would be easier to understand like this: `(?=\s*(?:\([^()]*\))?:)`.
    This also removes the capturing group (we want to avoid these if possible).

Reply:
Thank you for the suggestion, Michael! The new version reads better for sure. 👍 I also ran Gulp to include the minified version.